### DISCUSSION TITLE
Setup lantern demo : bulb light should be visible outside of lantern box

### DISCUSSION THREAD

**Discussion by arpitpolynine** [google/model-viewer#4731] [Q&A]
### Description

I am preparing lantern in which there is bulb inside and textured glass on the lantern wall. I want to show the light coming out from the bulb. To achieve this, I tried to give emissive material and lantern wall is Blend material. The light generated by bulb using emission should coming out from lantern wall. The lantern wall should enlighten using the bulb.

Current Material
Bulb = Emissive Material
Lantern wall = Opaque material

#### Live Demo

https://customer.polynine.com/lamp-demo/demo.html

![image_2024_03_29T12_55_45_430Z (1)](https://github.com/google/model-viewer/assets/58297022/edf3c25b-e41c-4c10-af5c-5694ca265873)

### Version

- model-viewer: v3.4.0

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [x] Edge
- [x] Firefox
- [x] IE
- [x] Safari

### OS

- [ x] Android
- [ x] iOS
- [ x] Linux
- [ x] MacOS
- [ x] Windows

**Comment by elalish** [comment]
This is getting pretty difficult for a rasterizer to handle - I don't know of any that support emissive textures illuminating other materials (a path tracer can do this, but for us, the performance just gets too bad). I'd recommend using the KHR_lights_punctual extension instead for the light bulb. See this sample model: https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/LightsPunctualLamp

**Reply by arpitpolynine** [reply]
Thank you for your help.

So modelviewer doesn't support KHR_lights_punctual extension. The given reference link is not looking like as open source. 

Can you help me where I can find a viewer which can support KHR_lights_punctual  and I can see how the lights are looking?

Thanks again, really appreciated!

**Reply by elalish** [reply]
model-viewer *does* support KHR_lights_punctual. And the link I gave *is* open. You can download that model and put it in MV to see the light rendering properly.
